### ISSUE TITLE
Tracking contributions beyond rustc

### ISSUE THREAD

**Issue by skade** [rust-community/team#36]
This is mainly a marker topic for me, nothing to act on immediately.

We go great length to give credit to anyone who submits patches to rustc, but theres far more people contributing to the Rust project, pretty directly even.

Maybe we should find a way to give credit to non-code contributions. I think Mozillas thing with Web Achievements was great (sadly defunct, as far as I see).

**Comment by steveklabnik** [comment]
I've been thinking about this for months. Big +1.

In the Rails world, we have the contributors app... Maybe we need a variant of it here?

**Comment by carols10cents** [comment]
Oooh yay! I think some of the things we talked about for the anniversary awards that I flaked on are relevant: 

- Non-Rust-team-members who have commits in Rust, Servo, Cargo, annnnddd.... a nursery crate? Their own crate? I'm trying to make something like [EGOT](https://en.wikipedia.org/wiki/List_of_people_who_have_won_Academy,_Emmy,_Grammy,_and_Tony_Awards) ;)
- Ran an event (those numbers are incredibly hard to collect, but hey)

If things like running an event are self-reported honors system, that could work. Other ideas:

- Documentation
- Bug reports
- Stack Overflow answers
